packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts

```diff
@@ -79,28 +79,34 @@ export const getStyles = memoizeFunction((
     },
 
     thumb: {
-      width: '.5em',
-      height: '.5em',
       borderRadius: '.5em',
       position: 'absolute',
       top: '.2em',
       transition: 'all 0.1s ease',
-      backgroundColor: thumbBackground,
+      borderColor: thumbBackground,
+      borderWidth: '.27em',
+      borderStyle: 'solid',
       left: '.2em'
     },
 
     thumbChecked: {
-      backgroundColor: thumbCheckedBackground,
+      borderColor: thumbCheckedBackground,
+      borderWidth: '.27em',
+      borderStyle: 'solid',
       left: '1.4em'
     },
 
     thumbDisabled: {
-      backgroundColor: thumbDisabledBackground,
+      borderColor: thumbDisabledBackground,
+      borderWidth: '.27em',
+      borderStyle: 'solid',
       left: '.2em',
     },
 
     thumbCheckedDisabled: {
-      backgroundColor: thumbCheckedDisabledBackground,
+      borderColor: thumbCheckedDisabledBackground,
+      borderWidth: '.27em',
+      borderStyle: 'solid',
```

Comment:
we don't need `borderWidth` and `borderStyle` on each selector but only on base `styles.thumb`. Just override the `borderColor` at other places like earlier.